stdlib/public/core/UnicodeScalar.swift

```diff
@@ -161,11 +161,11 @@ public struct UnicodeScalar :
     func lowNibbleAsHex(_ v: UInt32) -> String {
       let nibble = v & 15
       if nibble < 10 {
-        return String(UnicodeScalar(nibble+48))    // 48 = '0'
+        return String(describing: UnicodeScalar(nibble+48))    // 48 = '0'
```

Comment:
I don't think this expression should go through the reflection-based initializer.  String can be created from a `UnicodeScalar` in a lossless way.  (Same for `Character`.)